lib/index.js

```diff
@@ -185,7 +184,11 @@ class PluginPassportOAuth {
         }
         Object.keys(profile._json).forEach(attr => {
           if (this.config.strategies[profile.provider].persist.indexOf(attr) > -1) {
-            user[attr] = profile._json[attr];
+            if (this.config.strategies[profile.provider].kuzzleAttributesMapping) {
+              user[this.config.strategies[profile.provider].kuzzleAttributesMapping[attr]||attr] = profile._json[attr];
```

Comment:
nitpicking: the conditional key affectation is not easy to read here...

I'd rather write:
```js
const key = this.config.strategies[profile.provider].kuzzleAttributesMapping && this.config.strategies[profile.provider].kuzzleAttributesMapping[attr] || attr;
user[key] = profile._json[attr];
```